infra/root/terraform.tf

```diff
@@ -0,0 +1,22 @@
+terraform {
+  required_providers {
+
+    github = {
+      source  = "integrations/github"
+      version = "~> 6.0"
+    }
+  }
+
+  backend "s3" {
```

Comment:
I see this a few times, does this mean there are multiple state files, one for each environment? I'll be interested to learn how to manage these multiple environments.